test/index.js

```diff
@@ -0,0 +1,37 @@
+var scrape = require("readable-proxy").scrape;
+var path = require("path");
+var fs = require("fs");
+var prettyPrint = require("html").prettyPrint;
+var chai = require("chai");
+var chaiAsPromised = require("chai-as-promised");
+chai.should();
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+var testPageRoot = path.join(__dirname, "test-pages");
+var testPages = fs.readdirSync(testPageRoot).map(function(dir) {
+  return {
+    dir: dir,
+    source: path.join(testPageRoot, dir, "source.html"),
+    expected: path.join(testPageRoot, dir, "expected.html"),
+  };
+});
+
+describe("Test page", function() {
+  testPages.forEach(function(testPage) {
+    describe(testPage.dir, function() {
+      it("should render as expected", function() {
+        // Allows up to 10 seconds for parsing to complete.
+        // XXX: Scraping is damn slow. Investigate.
+        this.timeout(10000);
```

Comment:
Can you elaborate on why we need this scraping? I'm not as familiar with server-side JS, but is there a way to create a document from a local HTML file, and then run Readability on that directly?

Also, will errors in Readability.js propagate here? I could see us failing tests because of accidentally introducing JS errors, and it would be nice to be able to see those.